lib/Models/DynamicSegment.php

```diff
@@ -94,4 +107,16 @@ public static function bulkDelete($orm) {
 
     return ['count' => $count];
   }
+
+  /**
+   * @deprecated This is here for displaying the deprecation warning for properties.
+   */
+  public function __get($key) {
+    self::deprecationError('property "' . $key . '"');
+    return parent::__get($key);
```

Comment:
A very good idea to add this! I'm thinking about adding something similar also for `__callStatic` so that we cover also methods like `Model::findOne` etc. Do you think it is worth adding?

Reply:
Sure, I have added it.